src/pages/Extrinsic/components/ExtrinsicDetail.tsx

```diff
@@ -83,84 +81,67 @@ const ExtrinsicDetail: FC<{ className?: string }> = ({ className }) => {
             />
           )}
         </div>
-      </div>
-      <div className={'extrinsic-container'}>
+      </ExtrinsicContainer>
+      <ExtrinsicContainer>
         <Text
-          className={'grid-item_col1 '}
           color={'grey-500'}
         >
           Amount
         </Text>
         {/* TODO: due to API issues - amount of some transactions is object which is, for now, should be translated as zero */}
-        <div className={'grid-item_col11'}>
+        <div>
           <ChainLogo isInline={true} />
           {Number(amount) || 0} {chainData?.properties.tokenSymbol}
         </div>
         <Text
-          className={'grid-item_col1'}
           color={'grey-500'}
         >
           Fee
         </Text>
-        <div className={'grid-item_col11'}>
+        <div>
           <ChainLogo isInline={true} />
           {Number(fee) || 0} {chainData?.properties.tokenSymbol}
         </div>
-      </div>
-      <div className={'extrinsic-container'}>
+      </ExtrinsicContainer>
+      <ExtrinsicContainer>
         {hash && <><Text
-          className={'grid-item_col1'}
           color={'grey-500'}
         >
           Hash
         </Text>
-        <Text className={'grid-item_col11'}>
+        <Text>
           {deviceSize !== DeviceSize.sm ? hash : shortcutText(hash)}
         </Text></>}
         <Text
-          className={'grid-item_col1'}
           color={'grey-500'}
         >
           Extrinsic
         </Text>
-        <Text className={'grid-item_col11'}>{blockIndex}</Text>
-      </div>
-    </div>
+        <Text>{blockIndex}</Text>
+      </ExtrinsicContainer>
+    </>
   );
 };
 
-export default styled(ExtrinsicDetail)`
-  .extrinsic-container {
-    display: grid;
-    grid-column-gap: var(--gap);
-    border-bottom: 1px dashed #D2D3D6;
-    grid-template-columns: 85px 1fr;
-    font-size: 16px;
-    line-height: 20px;
-    padding: calc(var(--gap) * 1.5) 0 !important;
-    grid-row-gap: var(--gap);
-    &:nth-child(2) {
-      padding-top: 0 !important;
-    }
-  }
-
-  .container-with-border {
-    padding-bottom: calc(var(--gap) * 2);
-    border-bottom: 1px dashed #D2D3D6;
+const ExtrinsicContainer = styled.div`
```

Comment:
Sometimes you call it wrapper, sometimes container. we need to stick with some set of rules. I would prefer to just go with Wrapper everywhere, since container usually associates with smart components (pretty close to page)